templates/search_aggregates_v5.sql

```diff
@@ -0,0 +1,31 @@
+SELECT
+  submission_date,
+  app_version,
+  country,
+  distribution_id,
+  engine,
+  locale,
+  search_cohort,
+  source,
+  default_search_engine,
+  SUM(organic) as organic,
+  SUM(tagged_sap) as tagged_sap,
+  SUM(tagged_follow_on) as tagged_follow_on,
+  SUM(sap) as sap,
+  SUM(ad_click) as ad_click,
+  SUM(search_with_ads) as search_with_ads,
+  SUM(unknown) as unknown
+FROM
+  search_clients_daily_v7
```

Comment:
in python_mozetl `search_aggregates` groups by values per-ping, but `search_clients_daily` effectively only has values per-client because it only groups by [`client_id`, `submission_date`, `engine`, and `source`](https://github.com/mozilla/python_mozetl/blob/master/mozetl/search/aggregates.py#L55). It definitely makes this a lot simpler, but I'm not clear on whether that difference is acceptable. @harterrt or @wlach is this close enough?

Reply:
Also, the behavior in python_mozetl is completely deterministic for `search_aggregates`. If this is changed to match python_mozetl then we can validate by expecting the exact same results as the parquet import.

Reply:
I don't think it's quite accurate to say it's per-client, since it's highly likely that a client might submit multiple source/engine combinations per day.

I think this approach is almost certainly fine, and possibly better than what we're doing now (since there's a more intuitive mapping from search_aggregates back to search_clients_daily), though I would defer to @harterrt and/or @benmiroglio's expert opinion.

In either case, we should be testing this dataset side-by-side the existing parquet-based one for a while, in case there's something we're missing.

Reply:
Yes, this is definitely better. Thanks for making this simplification!

Regarding aggregating from client or ping: I believe `search_aggregates` only calculates search count `SUM`s so there shouldn't be a difference. As you mentioned, we should verify the resulting datasets are the same from both sources.

Just for context, I was actually just thinking through a similar change earlier this week. IIRC, these jobs were originally designed so that we could use the guts of the ETL for processing `main_summary` data in live analyses. That way if I wanted to see how (e.g.) search counts looked on two experiment branches I could use the same error handling and exploding logic for my live analysis. We'll still have to find a way to do this in the BQ world, but this ETL isn't the right place.